spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -148,9 +149,11 @@
     scenario "User tries to schedule publishing for date in the past" do
       given_there_is_a_draft_step_by_step_page
       and_I_visit_the_scheduling_page
+      then_inputs_should_have_tomorrows_date
       and_I_fill_in_the_scheduling_form_with_a_date_in_the_past
       when_I_submit_the_form
       then_I_should_see "Scheduled at can't be in the past"
+      but_inputs_should_retain_the_values_I_entered
```

Comment:
Can you rename this to `and_.....` rather than `but_....` so that the test steps follow the [Given
When Then](https://martinfowler.com/bliki/GivenWhenThen.html) pattern.

Reply:
['But' is part of the Given When Then pattern](https://cucumber.io/docs/gherkin/reference/#keywords), and makes it clearer that the date & time values should be retained _despite_ being invalid. Happy to rename `values` though 👍

Reply:
Can you also rename "inputs"? 
Personally, I think "and" is an easier read. E.g. "and_I_can_still_see_the_date_and_time_values_I_entered".